gulp/scripts.js

```diff
@@ -63,9 +63,14 @@ export function copyScripts() {
     .pipe(browserSync.reload({stream: true}))
 }
 
-export function lint() {
-  gulp.src([paths.scripts.input, `!${paths.scripts.dir}vendor/**/*`, `!${paths.scripts.dir}polyfills.js`])
+export function lint(done) {
```

Comment:
I would be good if the build failed before the selenium tests run. The fail fast approach should also help with build times.